### DISCUSSION TITLE
Allow manual sorting of presets

### DISCUSSION THREAD

**Discussion by emahuni** [directus/directus#19760] [Feature Requests]
### Summary

I noticed that presets are sorted alphabetically, I'd it'd be great if we could sort them manually.

### Basic Example

 Consider the following screenshot, I'd like them sorted based on how the order moves:

![image](https://github.com/directus/directus/assets/7290508/3c42ac0f-ed6d-4968-a411-7f5332aa95e0)

The desirable order would be:

- backlog
- preparing
- ready
- delivering
- settled
- cancelled

### Motivation

I want user to be able to look at the orders in the order of their statuses. Each preset should have its own place that is meaningful to the collection and usage of the preset.

### Detailed Design

This requires a sort field in the presets table and should be configurable in the preset dialog.

### Requirements List

__

### Drawbacks

__

### Alternatives

For now, the only way I can achieve this behaviour is to number the presets so that they are sorted according to the number assigned to them.
![image](https://github.com/directus/directus/assets/7290508/8172c5eb-c528-4099-87ce-ee022b980faf)

### Adoption Strategy

__

**Comment by ricardovigatti** [comment]
+1

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
